src/wui/game_debug_ui.cc

```diff
@@ -339,7 +281,10 @@ void FieldDebugWindow::think() {
 		}
 	}
 
-	ui_field_.set_text(str.c_str());
+	if (str.compare(text_) != 0) {
```

Comment:
```suggestion
	if (!text_.empty()) {
```

Reply:
Then the text would only be set during the first call, but will not update if the field properties change, while the window is open.

Reply:
I can never remember what 0 means in the string compare... `if (text_ == str)`? Strings have a direct equality operator these days, so there's no need for using the compare function.

Reply:
0 means equal, therefor `(str.compare(text_) != 0)` is the same as `(str != text_)`